releases/v0.8.0-rc0.yaml

```diff
@@ -1,6 +1,7 @@
 ---
 version: v0.8.0-rc0
 name: 0.8.0-rc0
+release-notes: Notes go here
```

Comment:
Why was this added?

Reply:
It's by this commit - https://github.com/submariner-io/releases/pull/57/commits/f288e34befb6045812348337f87e45b78bd604df